packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -163,19 +198,34 @@ async function downloadSimpleFinTransactions(acctId, since) {
     throw BankSyncError(res.error_type, res.error_code);
   }
 
-  const {
-    transactions: { all },
-    balances,
-    startingBalance,
-  } = res;
+  let accounts = res;
+  if (!batchSync) {
+    accounts = {
+      placeholder: res,
```

Comment:
I see why the `placeholder` thing is happening, but it feels a bit hackish.

Could you extract a method that gives the proper object for assignment, and then have something like an if (!batchSync) that does `retVal = thatFunction()` and then an else that does the for loop?